src/NGen.cpp

```diff
@@ -4,12 +4,14 @@
 #include <string>
 #include <thread>
 #include <unordered_map>
+#include <netcdf>
```

Comment:
This doesn't belong here. There's no need for core ngen to depend on NetCDF. It's even an optional library, not a hard requirement.